src/components/ArticleCards.jsx

```diff
@@ -0,0 +1,31 @@
+import Card from "react-bootstrap/Card";
+import SingleArticleCard from "./SingleArticleCard";
+import { Link } from "react-router-dom";
+
+export default function ArticleCards({ article }) {
+  return (
+    <>
+      <div className="article-card">
+        <Card className="item-card" style={{ width: "18rem" }}>
+          <Card.Img variant="top" src={article.article_img_url} />
```

Comment:
img need to have an alt attribute - needs to be relevant to the image. accessibility is very important